src/main/java/pl/javastart/task/Monitor.java

```diff
@@ -0,0 +1,25 @@
+package pl.javastart.task;
+
+public class Monitor {
+
+    String producer;
+    int sizeInches;
+    int refreshRate;
+
+    Monitor() {
+            }
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu